src/shared/fstab-util.c

```diff
@@ -34,6 +34,21 @@
 #include "strv.h"
 #include "util.h"
 
+bool fstab_has_fstype(const char *fstype) {
```

Comment:
Actually I lazily copied fstab_is_mount_point() and adapted it accordingly.

So I guess this function will need the same treatment.